test/gitea_test.go

```diff
@@ -303,7 +303,7 @@ func TestGiteaACLCommentsAllowing(t *testing.T) {
 			assert.NilError(t, err)
 
 			topts.PullRequest = tgitea.CreateForkPullRequest(t, topts, secondcnx, "", "echo Hello from user "+topts.TargetRefName)
-			topts.CheckForStatus = "success"
+			topts.CheckForStatus = "pending"
 			tgitea.WaitForStatus(t, topts, topts.PullRequest.Head.Sha)
```

Comment:
it will be in pending state when non authorized user send a PR
![Screenshot from 2023-05-30 15-46-01](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/dcb821d3-004e-4640-bf94-8254fcbdc7bf)

and when authorized user does /retest, /test it will go to success state
![Screenshot from 2023-06-12 15-00-36](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/72ca0a16-8ab4-462c-a6b6-9216b8ef48d4)

Reply:
so TestAcl don't tes the full thing anymore, just the pending state, can we get it to test the full flow?

Reply:
we  already doing that here https://github.com/openshift-pipelines/pipelines-as-code/pull/1310/files#diff-91d7f3778d1b7a7b41ae3ab7d064590ddaade6f8ec8cc29033a23f4a5dc111e9R312

initially its in pending state and when we do `/ok-to-test` or `/test` or `/retest` the expression goes to success and that we are checking

Reply:
ah yes you are right, if you do a new iteration would you add a comment of what the test do and that the pending state is excpeted here https://github.com/openshift-pipelines/pipelines-as-code/blob/765d5876cf7cc582b8fa8f6f5c24d0ca91112629/test/gitea_test.go#L253

generally adding comment on pr help getting the confusion out of the way before review and get them merged faster ;)

Reply:
Added comment to clear the test usage